clients/fides-js/src/components/ConsentButtons.tsx

```diff
@@ -14,22 +14,21 @@ const ConsentButtons = ({
   onManagePreferencesClick,
   onSave,
   enabledKeys,
+  isAcknowledge,
   isInModal,
 }: {
   experience: PrivacyExperience;
   onSave: (noticeKeys: NoticeKeys) => void;
   onManagePreferencesClick?: () => void;
   enabledKeys: NoticeKeys;
+  isAcknowledge: boolean;
   isInModal?: boolean;
 }) => {
   if (!experience.experience_config || !experience.privacy_notices) {
     return null;
   }
 
   const { experience_config: config, privacy_notices: notices } = experience;
-  const isAllNoticeOnly = notices.every(
```

Comment:
moved this up into `Overlay.tsx` so that we can send a value to `acknowledge_mode` in the patch notices-served request